src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String TITLE = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String report() {
+        StringBuilder result = new StringBuilder();
+        result.append(TITLE).append(System.lineSeparator());
+        for (Map.Entry<String, Integer> entry : Storage.fruits.entrySet()) {
+            result.append(entry.getKey())
+                 .append(SEPARATOR)
+                 .append(entry.getValue())
+                    .append(System.lineSeparator());
```

Comment:
```suggestion
                .append(System.lineSeparator());
```

Reply:
-_-" mnv said move this line to position 20